forge-gui/res/cardsfolder/upcoming/the_flux.txt

```diff
@@ -0,0 +1,11 @@
+Name:The Flux
+ManaCost:2 R R
+Types:Enchantment Saga
+K:Saga:6:DBDamage,DBExile,DBExile,DBExile,DBExile,DBMana
+SVar:DBDamage:DB$ DealDamage | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | NumDmg$ 4 | SpellDescription$ CARDNAME deals 4 damage to target creature an opponent controls.
+SVar:DBExile:DB$ Dig | Defined$ You | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ Exile the top card of your library. You may play that card this turn.
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ Play | SubAbility$ DBCleanup | ExileOnMoved$ Exile
+SVar:Play:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may play remembered card.
```

Comment:
I'd just use the last sentence of the SpellDesc here. "this turn" is probably good to add either way.